### PR TITLE
feat(Nanopore) Database model for Nanopore runs

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
Add model to store sequencing metrics for Nanopore data
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/store/models.py [hunk 1]
```diff
@@ -867,3 +868,23 @@ class SampleLaneSequencingMetrics(Model):
 
     def to_dict(self):
         return to_dict(model_instance=self)
+
+
+class NanoporeAcquisition(Model):
```
Comment (seallard) [inline cg/store/models.py:873]: What is an acquisition? Maybe NanoporeSequencingMetrics or similar is an alternative

**Review by seallard — APPROVED** [review]
Looks good! Maybe this merits some thought about how we can shape the architecture to easily integrate new types of machines and analyses in the future.

**Comment by Vince-janv** [comment]
> Looks good! Maybe this merits some thought about how we can shape the architecture to easily integrate new types of machines and analyses in the future.

Yes, I very much agree! I started a rough draft here: https://miro.com/app/board/uXjVN-I19i8=/ and I'm planning to discuss it in a refinement session with you guys 😊

File: cg/store/models.py (hunk 1 above)
Comment (Vince-janv) [reply cg/store/models.py:873]: Yes that was the alternative. Acquisition is what it's called elsewhere in the files and documentation (by the company ONT). It's basically a run/analysis. Let's see what the others prefer and go with that

**Comment by Vince-janv** [comment]
Outdated. New PR created

**Pull request closed without merging**